src/components/toaster/toast/toast.module.scss

```diff
@@ -0,0 +1,58 @@
+@import '@styles/variables.scss';
+
+.root {
+    position: absolute;
+    top: var(--headerHeight);
+    padding: 14px;
+    width: 480px;
+    color: var(--white);
+    pointer-events: all;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
+    gap: 14px;
+}
+
+.root.inCart {
+    inset: auto 0 0 auto;
+    width: 100%;
+    max-width: var(--drawerMaxWidth);
```

Comment:
?! Implicit dependency on a variable defined in a random component's file.

Remove this. Why does the toast component know about cart.

Reply:
Yes, I got this waking up at midnight, that cart can be quite different in projects and I should tell toast its location. Not only location. For e.g. `width` as well.